msprime/ancestry.py

```diff
@@ -1698,7 +1705,38 @@ class DiracCoalescent(ParametricAncestryModel):
 @dataclasses.dataclass
 class SweepGenicSelection(ParametricAncestryModel):
     """
-    .. todo:: Document me
+    This simulates a coalescent history with a single selective
```

Comment:
We're trying state what the model *is* rather than what its effect is. Can we summarise the model itself in a couple of sentences, rather than referring the reader to some papers? (Which are good, but we should "see XXX" for more details, rather than expecting the user to go off and read the paper)

Reply:
So, the comment would start with "A selective sweep in which ...". The first sentence should be enough to know what the model is in isolation, without any of the rest. Works like "This" are a waste of space, and don't work when we show the one-sentence summary in other contexts like summary tables.